mgz/summary/map.py

```diff
@@ -97,7 +97,7 @@ def lookup_name(map_id, name, version):
         elif version == Version.AOK:
             return name, False
         else:
-            raise ValueError('unspecified builtin map: ' + str(map_id))
+            name = 'Unknown Map ' + str(map_id)
```

Comment:
Similar to my comment on the unknown actions, an unknown map is an actual parsing issue that deserves to raise an exception. The way to adapt the code is to figure out what the unknown map is and add it to the consts. The name of the map can be obtained by playing back the match or by printing out the `instructions` text via the parser.